src/fastgltf_parser.hpp

```diff
@@ -321,14 +313,29 @@ namespace fastgltf {
          * Loads the file with a optional byte offset into a memory buffer.
          */
         bool loadFromFile(const std::filesystem::path& path, std::uint64_t byteOffset = 0) noexcept;
-
+        
         /**
          * Returns the size, in bytes,
          * @return
          */
         [[nodiscard]] inline std::size_t getBufferSize() const noexcept;
     };
 
+    #if defined(__ANDROID__)
+    class AndroidGltfDataBuffer : public GltfDataBuffer {
+        AAssetManager* asset_manager = nullptr;
```

Comment:
This is a redundant initializer given that the only constructor of `AndroidGltfDataBuffer` writes a value to `asset_manager`. Btw, lowerCamelCase please?